CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.7)
 project(xunused)
 
+set(CMAKE_FIND_PACKAGE_SORT_ORDER NATURAL)
```

Comment:
Could you please add a comment to why this was added for future reference?

Reply:
Thanks for the tip, I've done it.